doc/ref/mloop.xml

```diff
@@ -161,6 +161,24 @@ Var="time"/>. This is an integer that holds the number of
 milliseconds. Similarly the amount of memory allocated during that
 statement (in bytes) is stored in the variable <Ref Var="memory_allocated"/>.
 
+The variables <C>last</C>, <C>last2</C>, <C>last3</C>, <Ref
+Var="time"/> and <Ref Var="memory_allocated"/> are all write-protected
+by default. They can be changed using the function <Ref Func="UpdateStat"/>.
+
+<ManSection>
+<Heading>UpdateStat</Heading>	
+<Func Name="UpdateStat" Arg='name, value'/>
+
+<Description>
+<Ref Func="UpdateStat"/> assigns the variable with name <A>name</A> the
+value <A>value</A>. Valid values for <A>name</A> in this version of GAP are
+<C>"last"</C>, <C>"last2"</C>, <C>"last3"</C>, <C>"time"</C> and
+<C>"memory_allocated"</C>. This does not disable these values being
+automatically updated at the end of each executed statement.
```

Comment:
Perhaps like this?
> This does not disable the automatic updating of these variables performed by the interpreter,
> e.g. in interactive sessions.

Also, I think it would be a good idea to add a blurb that explains why one would want to use this function (namely for demos, and ideally one should not use it for anything else...

That said, I am not even sure that we really should document this function. Right now, demo code needs is very low level and needs to access various other low level and undocumented functions. Perhaps it would be better to hold off documenting this, until we had time to revise the Demo code -- if done right, this might get rid of the need for 3rd parties to roll their own variants of that. After all, this is a rather low level aspect of the system, and we don't document many other aspects of it, e.g. `SHELL`...

Reply:
I all-caps it, and moved a (briefer) version of the docs inline